### DISCUSSION TITLE
⏳ 🔗 Repeater Field for Elementor Pro Form

### DISCUSSION THREAD

**Discussion by davidefiorio** [elementor/elementor#16463] [Feature Request]
Only do I need a new function in Elementor Form?
I'm refear to a group fields that I can add in front-end with a button "add new row" to add and repeat some field.
Similar to "booking form" person1 ... person2... "add new person" and so on...

Tnx
Davide from Italy

**Comment by nerds-farm** [comment]
Hi Davide,

there is a third part integration to resolve your request:
https://e-addons.com/elementor/repeater-field/

**Comment by terracellc** [comment]
While the add would work - this seems like functionality that should be present and would give Elementor a big competitive advantage in the space.

**Reply by carry2web** [reply]
I second this Idea. Please add repeater to form

**Reply by web-orbit12** [reply]
I second this idea too.

**Comment by vgstef** [comment]
I also need this feature!

**Comment by whoaloic** [comment]
I support this feature :)

**Comment by pwnoobz** [comment]
How do we get this rolling?

**Comment by Henzxon** [comment]
If you don't mind me adding to this. I think it would work really well with `conditional fields` 🚀

**Comment by saigonnv** [comment]
You can try the add-on: https://add-ons.org/plugin/elementor-forms-repeater-fields/

**Reply by CapEric** [reply]
are you from VietNam?

**Comment by web-orbit12** [comment]
it's been 3 years...

**Comment by Post79** [comment]
Is this feature part of the planned "Advanced Form Builder Features" on the roadmap?
@nicholaszein
